clients/rust/src/lib.rs

```diff
@@ -1,4 +1,20 @@
 mod generated;
 
+use {
+    bytemuck_derive::{Pod, Zeroable},
+    solana_program::pubkey::Pubkey,
+};
+
 pub use generated::programs::SOL_STAKE_VIEW_PROGRAM_ID as ID;
 pub use generated::*;
+
+/// Helper struct to easily handle the return data created by the
+/// `GetStakeActivatingAndDeactivating` instruction.
+#[repr(C)]
+#[derive(Clone, Copy, Debug, Default, PartialEq, Pod, Zeroable)]
+pub struct GetStakeActivatingAndDeactivatingReturnData {
+    pub delegated_vote: Pubkey,
```

Comment:
You could use the `PodOption<Pubkey>` type here so it is easier for a client to check whether there is a delegation or not.

Reply:
Ah nice idea, done! It's a tiny annoyance that `PodOption` doesn't implement `Default`, but otherwise all good